manifests/server.pp

```diff
@@ -70,6 +72,7 @@
   $install_options         = undef,
   $install_secret_file     = $mysql::params::install_secret_file,
   $manage_config_file      = $mysql::params::manage_config_file,
+  $options                 = {},
```

Comment:
It would be nice if you could add a datatype here that enforces that this is a hash. Also you can enforce the types within the hash

Reply:
I added the `mysql::options` data type, is that what you were thinking about?

Reply:
Where is the datatype? I'm unable to find it in your branch.

Reply:
@bastelfreak Sorry, I forgot to push the changes. They are pushed now.